lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -328,6 +328,11 @@ static Value buildUnitNormalCdf(OpBuilder &b, Location &loc, Value x) {
   return buildNormalCdf(b, loc, x, zero, one);
 }
 
+// These constants control the reduction behavior of the loss functions.
+// None, Mean and Sum corresponds to "do not reduce", "Mean of losses", and "sum
+// of losses" respectively.
+enum Reduction { None, Mean, Sum, END };
```

Comment:
we can move this definition to `include/torch-mlir/Dialect/Torch/Utils/Utils.h` since it's shared by multiple files. Also, please include the `aten/src/ATen/core/Reduction.h` in the comments for future reference.

Reply:
Would the file https://github.com/llvm/torch-mlir/blob/main/include/torch-mlir/Dialect/Torch/Utils/TorchUpstream.h be a better location for this `enum`, since it is code from torch upstream?

Reply:
yea, you are right. that file should be a better place.